cmd/proxy/actions/app_proxy.go

```diff
@@ -2,16 +2,20 @@ package actions
 
 import (
 	"github.com/gobuffalo/buffalo"
+	"github.com/gomods/athens/pkg/download"
 	"github.com/gomods/athens/pkg/module"
 	"github.com/gomods/athens/pkg/storage"
 )
 
 func addProxyRoutes(app *buffalo.App, storage storage.Backend, mf *module.Filter) error {
 	app.GET("/", proxyHomeHandler)
-	app.GET("/{module:.+}/@v/list", listHandler(storage))
-	app.GET("/{module:.+}/@v/{version}.info", cacheMissHandler(versionInfoHandler(storage), app.Worker, mf))
-	app.GET("/{module:.+}/@v/{version}.mod", cacheMissHandler(versionModuleHandler(storage), app.Worker, mf))
-	app.GET("/{module:.+}/@v/{version}.zip", cacheMissHandler(versionZipHandler(storage), app.Worker, mf))
+
+	// Download Protocol
+	app.GET(download.ListPath, download.ListHandler(storage, proxy))
```

Comment:
not complaining, just an idea can we start names of these constant with Path?
just like we do with errors: **Err**MyNewError
so when I code I type Err and get all the errors in the package, here i would type Path and got the list of the paths

Reply:
it feels weird but ok at the same time